README.md

```diff
@@ -1243,3 +1243,78 @@ out  | evt.ota_end.report      | object     | Sent on upgrade end with upgrade s
    }
 }
 ```
+
+### Zwave Specific service 
+This service is used to meet all zwave specific requirements that are not satisfied with already implemented services.
+
+#### Service name 
+
+`zwave_specific`
+
+#### Interfaces
+
+Type | Interface               | Value type | Description 
+-----|-------------------------|------------|--------------
+out  | evt.notification.report | str_map    | Notification Reports that are sent by slave devices.
```

Comment:
nit: change to `Notification reports that are sent by worker devices.` 

I changed slave to worker, but if slave is used in the zwave documentation, we should stick with that.

Reply:
From zwave perspective "slave" means a device that is added to a so-called "controller" and cannot operate itself with network's capabilities. So I think we can stick with that.

Reply:
Sounds good. Still change it to `Notification reports that are sent by slave devices.` :)